includes/Core/Key_Metrics/Key_Metrics_Setup_Completed.php

```diff
@@ -30,10 +30,11 @@ class Key_Metrics_Setup_Completed extends Setting {
 	 * Gets the expected value type.
 	 *
 	 * @since 1.108.0
+	 * @since n.e.x.t Type changed to integer.
 	 *
 	 * @return string The type name.
 	 */
 	protected function get_type() {
-		return 'boolean';
```

Comment:
The IB also calls for changing the setting used for this to `key_metrics_setup_completed_by_user_id`.

Reply:
@aaemnnosttv It seems I misunderstood this part, will change the name as well